web-console/knikubevirt/deplying-kubevirt/deploying-kubevirt.md

```diff
@@ -0,0 +1,49 @@
+# Deploying Kubevirt on OCP using UI
+
+## Pre-Installation
+
+![BasSeaching for HCO on the OperatorHub](img/1-01.png)
+On the OperatorHub, get a match for searching “KubeVirt HyperConverged/CNV/virt/virtualization…” etc (we currently have only a hit with “Hyperconverged…”)
+
+![Installing Operator](img/1-02.png)
+After viewing details, including a list of the APIs, the user clicks ‘Install’
```

Comment:
Oh, I did forget to place them here.
But looking at it again I don't think that reviewing the list of APIs on this Operator summary:
- We do install them all anyway, the user doesn't have a say, it's all or no installation.
- Because of that, I think that viewing the APIs, anywhere on the installation process, is a cognitive distraction
- We abstract them from the user on the installation. The user does actually see them via the CR list on the second step of the wizard, but the APIs are not the subject there, just a means to display the CR list and customize them.

WDYT?

Reply:
Yeah I'd agree we probably don't need to take the space to show them again here!